channels/us.m3u

```diff
@@ -5709,8 +5703,6 @@ https://dai.google.com/linear/hls/event/7_v7qMjnQWGZShy2eOvR5g/master.m3u8
 https://dai.google.com/linear/hls/event/IG9ThaPaTwCojeoEWVNZRQ/master.m3u8
 #EXTINF:-1 tvg-id="TennisChannelHD.us" tvg-name="Tennis Channel HD" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/wcmQ30R.png" group-title="Sports",Tennis Channel HD
 https://tennischannel-intl-samsung-uk.amagi.tv/playlist.m3u8
-#EXTINF:-1 tvg-id="TennisChannelPlus2.us" tvg-name="Tennis Channel+ 2" tvg-country="US" tvg-language="English" tvg-logo="http://tennischanneleverywhere.com/resources/assets/tennischanneleverywhere/images/logos/tcplus-logo-1x-01.png" group-title="Sports",Tennis Channel+ 2
-https://playoutengine.sinclairstoryline.com/playout/e1838c44-dcb6-47b5-93d4-a0547833518f/g.m3u8
```

Comment:
@MapGuy11 will need to test that one. I have 403 Forbidden on my side.

Might be geo-locked, but since I don't have a VPN for now, can't really see.

Reply:
Works for me with proxies but shows a single frame because there's no live events as of now

Reply:
![image](https://user-images.githubusercontent.com/43685307/125862294-651a0849-ac2a-4321-91ae-b1887b5e9692.png)

Reply:
Also tested the other ones, same message. All the Tennis channels are now Geo-blocked.